code-challenges.test.js

```diff
@@ -35,9 +62,35 @@ const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+// describe("oddsOnly", () => {
+//   it("returns an array of only odd numbers from least to greatest", () => {
+//     expect(oddsOnly("array of numbers")).toEqual("array of only odd numbers in order")
 
 // b) Create the function that makes the test pass.
 
+// Pseudo Code:
+// declare a function oddsOnly
+// set the parameter to an array
+// itterate and filter through the array
+// check if a number
+// if number, check if odd with modulo
+// if odd, include in new array
+// if even, omit from new Array
+// return sorted new array from least to greatest
+
+const oddsOnly = (array) => {
+  let newArray = array.filter(
+    value => value % 2 !== 0 && typeof(value) === 'number'
+  )
+  return newArray.sort(((a,b)=>a-b))
```

Comment:
Looks like there is one extra set of parenthesis that came along for the ride

Reply:
oops, but it looked like the code still worked?